src/App.tsx

```diff
@@ -33,77 +64,16 @@ export const App: React.FC = () => {
       </div>
 
       {/* Move this markup to Pagination */}
```

Comment:
There is an empty block `{}` that seems to serve no purpose. If it's not needed, it should be removed to clean up the code.

Reply:
Ok